src/ResourceManager/KeyVault/Commands.KeyVault.Test/packages.config

```diff
@@ -4,20 +4,21 @@
   <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Gallery" version="2.6.2-preview" targetFramework="net45" />
-  <package id="Microsoft.Azure.KeyVault" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Authorization" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.KeyVault" version="1.0.1" targetFramework="net45" />
+  <package id="Microsoft.Azure.KeyVault" version="2.0.1-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.KeyVault.WebKey" version="2.0.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Authorization" version="2.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.KeyVault" version="2.0.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Resources" version="2.20.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Test.Framework" version="1.0.6052.28118-prerelease" targetFramework="net45" />
   <package id="Microsoft.Azure.Test.HttpRecorder" version="1.6.7-preview" targetFramework="net45" />
-  <package id="Microsoft.Bcl" version="1.1.9" targetFramework="net45" />
+  <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net45" />
   <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net45" />
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.1" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
   <package id="Microsoft.Rest.ClientRuntime" version="2.3.2" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.2" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure.Authentication" version="2.2.8-preview" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.1" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure.Authentication" version="2.2.6-preview" targetFramework="net45" />
   <package id="Microsoft.Rest.ClientRuntime.Azure.TestFramework" version="1.3.4-preview" targetFramework="net45" />
   <package id="Microsoft.WindowsAzure.Management" version="4.1.1" targetFramework="net45" />
```

Comment:
Seems to be way outdated. This is the management NuGet "Microsoft.Azure.Management.ResourceManager"